src/hooks/useUserAssetsBalance.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
```

Comment:
doesn't look like this hook is used anywhere?

Reply:
https://github.com/rainbow-me/rainbow/pull/5959#discussion_r1693415120

Yeah left it because could be useful in the future.